rights

```diff
@@ -1,5 +1,34 @@
-# -*- mode: conf -*-
-# vim:ft=cfg
+# Allow all rights for the Administrator
```

Comment:
please comment out all as "empty" should be default. Also shift below existing examples, but add a header like "for use with LDAP authentication"